### DISCUSSION TITLE
Http表单批量发送byte数组数据咨询

### DISCUSSION THREAD

**Discussion by sowov** [looly/hutool#3492] [Q&A]
在接收多个文件数据后，如何批量发送byte数组，是否只能先落盘成文件，再通过  HttpRequest form(String name, File... files)  此方法去实现，是否有不落盘的做法，减少IO次数。

**Comment by looly** [comment]
那你只能通过byte[]方式存储，或者使用HttpUrlconnecton获取原生流，通过流拷贝方式写出。
